pages/contact.vue

```diff
@@ -5,35 +5,26 @@
       :description="$t('contact.description')"
     />
 
-    <div class="max-w-7xl mx-auto px-4 sm:px-6 text-center relative z-10 pb-32">
-      <ul class="grid grid-cols-1 md:grid-cols-3 gap-6">
+    <div class="max-w-7xl mx-auto px-4 sm:px-0 text-center relative z-10 pb-32">
+      <ul class="grid grid-cols-1 md:grid-cols-2 gap-10">
         <li
           v-for="item in options"
-          :class="item.slug === 'other' && 'md:col-span-3'"
-          class="border border-slate-300 px-4 py-8 rounded-lg flex flex-col justify-between items-center"
+          class="border-2 border-white px-12 py-[3.75rem] rounded-lg flex flex-col justify-between items-center box"
         >
           <div>
             <component
               :is="item.icon"
               class="h-16 w-16 mx-auto"
               aria-hidden="true"
             />
-            <h2 class="text-2xl sm:my-2">{{ item.title }}</h2>
-            <p class="mb-6">{{ item.description }}</p>
+            <h2 class="text-[2.5rem] leading-[3.75rem] sm:my-4">{{ item.title }}</h2>
```

Comment:
```suggestion
            <h2 class="text-2xl sm:text-4xl my-4">{{ item.title }}</h2>
```
Please adjust the font size for mobile.

Reply:
resolved with [7ad6314](https://github.com/AstarNetwork/astarwebsite_v2/pull/354/commits/7ad63144e7cea367f549e60a6ef808ab59c16b97)